api/route/base.go

```diff
@@ -37,19 +38,20 @@ func SetUpRouter() *gin.Engine {
 	r := gin.New()
 	store := cookie.NewStore([]byte("secret"))
 	r.Use(sessions.Sessions("session", store))
-	r.Use(filter.CORS(), filter.Authentication(), filter.RequestId(), filter.RecoverHandler())
+	r.Use(filter.CORS(), filter.Authentication(), filter.RequestId(), filter.RecoverHandler()) //
 
 	AppendHealthCheck(r)
 	AppendAuthentication(r)
 	//AppendRoute(r)
 	AppendSsl(r)
 	AppendPlugin(r)
 	AppendUpstream(r)
-	AppendConsumer(r)
+	//AppendConsumer(r)
```

Comment:
just removed the extra comment line

Reply:
No, just keep this until all API migrating completed.
This can help us compare the previous code